daprdocs/content/en/reference/components-reference/supported-cryptography/local-storage.md

```diff
@@ -0,0 +1,52 @@
+---
+type: docs
+title: "Local storage"
+linkTitle: "Local storage"
+description: Detailed information on the local storage cryptography component
+---
+
+## Component format
+
+The purpose of this component is to load keys from a local directory.
+
+The component accepts as input the name of a folder, and loads keys from there. Each key is in its own file, and when users request a key with a given name, Dapr loads the file with that name.
+
+Supported file formats:
+
+- PEM with public and private keys (supports: PKCS#1, PKCS#8, PKIX)
+- JSON Web Key (JWK) containing public, private, or symmetric keys
```

Comment:
```suggestion
- JSON Web Key (JWK) containing a public, private, or symmetric key
```

Reply:
This is because it accepts a single JWK. For more than one key, use the JWKS component.